app/views/unattended/provisioning_templates/PXEGrub/pxegrub_default_local_boot.erb

```diff
@@ -4,8 +4,16 @@ name: PXEGrub default local boot
 model: ProvisioningTemplate
 %>
 <%# Used to boot provisioned hosts, do not associate or change the name. %>
-
-default=0
+<%
+  # Grub1 only supports numeric default statement, this allows conversion to number. First define
+  # array of directories we will search for EFI booloaders:
+  paths = ["fedora", "redhat", "centos", "debian", "ubuntu", "sles", "opensuse", "Microsoft", "EFI"]
+  # Read default setting but since "local" is missing, use the first path available.
+  default_setting = global_setting("default_pxe_item_local", paths.first)
+  default_setting = paths.first if default_setting == 'local'
+  default_item = items.index(default_setting) || 0
```

Comment:
I'm sorry, but still on more thing. I was getting:

```
Warning! There was an error rendering the PXEGrub default local boot template: Safemode doesn't allow to access 'items' on #<Safemode::ScopeObject>
```
Due to this line. It should be `s/items/paths/`

Reply:
Damn, template snapshots will help to catch these, it's another PR.